src/lib/comms/include/sol-netctl.h

```diff
@@ -309,6 +375,146 @@ int32_t sol_netctl_service_get_strength(
     const struct sol_netctl_service *service);
 
 /**
+ * @brief Gets the service favorite
+ *
+ * This function gets the service favorite for the netctl service,
+ * after the service is to give by service monitor.
+ *
+ * @param service The service structure which the service favorite is
```

Comment:
This sentence is quite diffucult to understand. I would change to something like "The service struct which will be used to check if it's the favorite" or something similar